package.json

```diff
@@ -591,17 +591,22 @@
     "vscode:prepublish": "yarn compile",
     "compile": "tsc -p ./",
     "watch": "tsc -watch -p ./",
-    "test": "yarn compile",
+    "pretest": "npm run compile",
+    "test": "mocha --ui tdd \"./out/test/*.js\"",
     "build": "vsce package --yarn",
     "vscode:publish": "vsce publish --yarn",
     "ovsx:publish": "ovsx publish",
     "format": "prettier --write '**/*.{ts,js,json,yml}'",
     "format-check": "prettier --check '**/*.{ts,js,json,yml}'"
   },
   "devDependencies": {
+    "@types/glob": "^7.1.3",
+    "@types/mocha": "^8.2.2",
     "@types/node": "15.12.2",
     "@types/shell-quote": "1.7.0",
     "@types/vscode": "1.43.0",
+    "glob": "^7.1.7",
+    "mocha": "^9.0.0",
```

Comment:
minor, but I would prefer using `jest` since we already use it in metals-languageclient.

Reply:
I implemented using jest, can you double check everything looks ok? 
It's the first time I configured it!

Reply:
Looks great, thanks!